docs/tutorials/projects/pet-treat-dispenser.md

```diff
@@ -71,7 +71,7 @@ The front of the main body of your print is the side with the dog bone.
 
 {{<gif webm_src="../../img/pet-treat-dispenser/3d-print-design.webm" mp4_src="../../img/pet-treat-dispenser/3d-print-design.gif" alt="Rotating 3d rendered model of the pet feeder design." class="alignright" max-width="300px" class="aligncenter">}}
 
-2. Mount your Raspberry Pi to the side of the main body of your pet feeder using the provided mounting screw holes.
+1. Mount your Raspberry Pi to the side of the main body of your pet feeder using the provided mounting screw holes.
```

Comment:
```suggestion
2. Mount your Raspberry Pi to the side of the main body of your pet feeder using the provided mounting screw holes.
```

^ this is on purpose so that it continues counting from 2 here

Reply:
indenting the gif shifts it slightly off center